buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateEapBuildNumber.kt

```diff
@@ -47,7 +47,7 @@ open class UpdateEapBuildNumber : DefaultTask() {
     val properties = IntellijVersionHelper.getProperties()
     val latestEapBuildNumber = IntellijVersions.eapOfLatestSupportedMajor
```

Comment:
Aaand remove this one now (btw, Ctrl+Alt+N to inline a value, at least on Linux)